src/goto-analyzer/static_show_domain.h

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************\
+
+Module: 
+
+Author: Martin Brain, martin.brain@cs.ox.ac.uk
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_ANALYZER_STATIC_SHOW_DOMAIN_H
+#define CPROVER_GOTO_ANALYZER_STATIC_SHOW_DOMAIN_H
+
+#include <iosfwd>
+
+#include <util/message.h>
+#include <util/namespace.h>
+#include <util/options.h>
```

Comment:
See elsewhere: I'd opt for forward declarations; though namespace.h certainly is redundant.